### PR TITLE
[O2B-851] allow only admin to create and delete tags

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- only admin is authorised to create and delete tags

Notable changes for developers:
- NA

Changes made to the database:
- NA

### CODE REVIEW

**Review by graduta — CHANGES_REQUESTED** [review]
Please update the tests accordingly and add new tests in the API folder

File: lib/server/routers/tags.router.js [hunk 1]
```diff
@@ -32,7 +32,8 @@ module.exports = {
         },
         {
             method: 'post',
-            controller: TagsController.createTag,
+            controller: [rbacMiddleware([BkpRoles.ADMIN]), TagsController.createTag],
+            args: { public: true },
```
Comment (graduta) [inline lib/server/routers/tags.router.js:36]: Why did you add `public true`?

**Review by graduta — CHANGES_REQUESTED** [review]

File: lib/server/routers/tags.router.js (hunk 1 above)
Comment (xsalonx) [reply lib/server/routers/tags.router.js:36]: by accident
